internal/promotion/promotion.go

```diff
@@ -165,13 +167,19 @@ func (s *Step) GetErrorThreshold(runner promotion.StepRunner) uint32 {
 func (s *Step) BuildEnv(promoCtx Context, opts ...StepEnvOption) map[string]any {
 	env := map[string]any{
 		"ctx": map[string]any{
-			"project":   promoCtx.Project,
-			"promotion": promoCtx.Promotion,
-			"stage":     promoCtx.Stage,
+			"project":       promoCtx.Project,
+			"promotion":     promoCtx.Promotion,
+			"stage":         promoCtx.Stage,
+			"targetFreight": promoCtx.TargetFreightRef.Name,
```

Comment:
Should this be split out from the `targetFreight` section under `meta`? Feels weird to have the freight name up here, but warehouse info under `meta`.

Reply:
This follows the same pattern as promotion.actor did when it was added. It appears to be the desired behavior by project maintainers. I am happy to change it otherwise 😇